doc/source/release-notes/Percona-Server-5.6.36-82.1.rst

```diff
@@ -0,0 +1,62 @@
+.. rn:: 5.6.36-82.1
+
+============================
+|Percona Server| 5.6.36-82.1
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.36-82.1 on
+July 28, 2017 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.36-82.1/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.36
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-36.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.34-79.1 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.36-82.1
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.36-82.1>`_.
+
+New Features
+============
+
+ 
+
+Bugs Fixed
+==========
+
+ |Percona Server| could crash when running a data lookup on the partition table
+ if there was only one partition over the time period following simple test. Bug
```

Comment:
Looking at the commit messages, this is probably "A query over a partitioned table that uses an index to read a range of rows could crash if this range was not covered by any existing partition", ping @percona-ysorokin